lib/filemanager.js

```diff
@@ -334,17 +334,36 @@
             }
 
             var targetFullPath = path.resolve(basePath, targetRelativePath);
-            return this._moveFileHelper(sourceFullPath, targetFullPath);
+            return this._moveFileHelper(sourceFullPath, targetFullPath)
+                .thenResolve(targetFullPath);
+        }.bind(this));
+    };
+
+    /**
+     * Move a file at a given absolute path into the base directory managed by this
+     * FileManager instance to the given relative path.
+     *
+     * @param {string} sourceFullPath
+     * @param {string} targetFullPath
+     * @return {Promise.<string>} Resolves with the full target file path when complete
+     */
+    FileManager.prototype.moveFileAbsolute = function (sourceFullPath, targetFullPath) {
+        return this._later(function () {
+            return this._moveFileHelper(sourceFullPath, targetFullPath)
+                .thenResolve(targetFullPath);
         }.bind(this));
     };
 
     /**
      * Move a file under the base directory managed by this FileManager instance
      * to the given relative path.
+     *
```

Comment:
Ok, will do.  I wanted you to at least make sure you guys weren't using it in the crema code (as a node dep) that I wasn't finding.